src/ufm/countermeasures.py

```diff
@@ -3,6 +3,8 @@
 import torch
 import torch.nn as nn
 
+logger = logging.getLogger()
```

Comment:
`logging.getLogger(__name__)` again

Reply:
Also, this probably should _not_ be in the top level of this file and instead stored as a variable in each class on init